### PR TITLE
adding support for circleci

### LINKED ISSUE #81: Migrating from TravisCi to Circle CI
TravisCi doesnot pass test cases occasionally and switching to CircleCi would be better option!

### PR DESCRIPTION
**Pull request by SIMRAN1**
Fixes #81 
Adding support for circleCi and removing Travis CI

### CODE REVIEW

**Comment by ghost** [comment]
Hi @SIMRAN1!

It looks like one or more of your builds have failed.
I've added the relevant info below to save you some time.

- [Build 156.1](https://travis-ci.org/fossasia/open-event-ios/jobs/390306053) has `failed`

**Comment by SIMRAN1** [comment]
@imjog please review!!

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+          command: sh ./Scripts/build_for_testing.sh
+
+ 
+          
+workflows:
+  version: 2
+  build_and_test:
+    jobs:
+      - build-and-test
+
```
Comment (jogendra) [inline .circleci/config.yml]: remove the last empty line (23)

**Review by jogendra — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (SIMRAN1) [reply .circleci/config.yml]: okk @imjog !!

**Comment by jogendra** [comment]
@mariobehling @chashmeetsingh Please integrate the Cirlce-CI to the repo and remove Travic-CI integration.

**Comment by SIMRAN1** [comment]
Done @imjog !!

**Comment by jogendra** [comment]
@SIMRAN1 Remove end-space from the last line too.

**Review by jogendra — APPROVED** [review]
Nice!

**Comment by SIMRAN1** [comment]
thanks @imjog !!

**Pull request merged**